code/game/machinery/Sleeper.dm

```diff
@@ -279,3 +279,25 @@
 
 /obj/machinery/sleeper/old
 	icon_state = "oldpod"
+
+/obj/machinery/sleeper/party
+	name = "party pod"
+	desc = "'Sleeper' units were once known for their healing properties, until a lengthy investigation revealed they were also dosing patients with deadly lead acetate. This appears to be an old 'sleeper' unit repurposed as a 'Party Pod'. It’s probably not a good idea to use it."
+	icon_state = "partypod"
+	idle_power_usage = 3000
+	circuit = /obj/item/circuitboard/machine/sleeper/party
+	ui_x = 260
+	ui_y = 375
+
+	controls_inside = TRUE
+	possible_chems = list(
+		list(/datum/reagent/consumable/ethanol/beer, /datum/reagent/consumable/laughter),
+		list(/datum/reagent/spraytan),
+		list(/datum/reagent/colorful_reagent),
+		list(/datum/reagent/drug/space_drugs)
+	)//Exclusively uses non-lethal, "fun" chems. At an obvious downside.
+	enter_message = "<span class='notice'><b>You're surrounded by some funky music inside the chamber. You go deaf as you feel waves of krunk vibe within you.</b></span>"

```

Comment:
shouldn't this make you actually deaf while in the pod, then?

Reply:
Its mimicking the OG sleeper flavor text that said something about senses going dull

Reply:
Mm, I see what your saying from a consistency standpoint though. I'll throw in a quick reword.